metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -549,6 +571,10 @@ entities:
     keyAspect: dataHubPersonaKey
     aspects:
       - dataHubPersonaInfo
+  - name: dataHubAction
```

Comment:
Do we need the other aspects, or I guess that will come in a followup?

Reply:
don't need other aspects - this is required to allow `source` in metadataAttribution to contain a dataHubAction urn.